tdrs-backend/tdpservice/search_indexes/admin.py

```diff
@@ -0,0 +1,130 @@
+"""ModelAdmin classes for parsed data files."""
+
+from django.contrib import admin
+from .models import T1, T2, T3, T4, T5, T6, T7
```

Comment:
@jtimpe for awareness, T1-T7 record types are specific to the TANF Data Report only. this does not map to the SSP-MOE Data Report, though it is similar. 

Also worth mentioning. TANF and Tribal TANF data reports are almost identical, except for T2 records. Tribal TANF does not include one data element. I'll flag it below for reference.